### PR TITLE
Release/2.0.0

### PR DESCRIPTION
**Pull request by dawiddylowicz**
Cross-device release (CX-1377)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,12 +1,14 @@
 {
   "name": "onfido-sdk-ui",
-  "version": "1.1.0",
+  "version": "2.0.0-rc.1",
```
Comment (dawiddylowicz) [inline package.json]: This will be bumped to 2.0.0 before merging

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-242-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://242-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-242-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://242-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-242-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://242-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-242-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-242-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://242-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-242-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-242-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://242-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://242-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-242-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://242-pr-onfido-sdk-ui-onfido.surge.sh

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,11 +1,24 @@
...
 This change log file is based on best practices from [Keep a Changelog](http://keepachangelog.com/).  
 This project adheres to [Semantic Versioning](http://semver.org/). Breaking changes result in a different MAJOR version. UI changes that might break customizations on top of the SDK will be treated as breaking changes too.  
 This project adheres to the Node [default version scheme](https://docs.npmjs.com/misc/semver).  
 
-## [next-version]
+## [2.0.0]
+
+**Note:**
+* This version is not backwards-compatible. Migration notes can be found in [MIGRATION.md](MIGRATION.md)
+
```
Comment (hjheath) [inline CHANGELOG.md]: I think we should mention that we added cross device flow

File: release_guidelines.md [hunk 3]
```diff
@@ -34,7 +38,30 @@ An example `<version>` could be `2.0.0`
...
+* See if these exist:
+  * `https://raw.githubusercontent.com/onfido/onfido-sdk-ui/<tag>/dist/style.css`
+  * `https://raw.githubusercontent.com/onfido/onfido-sdk-ui/<tag>/dist/onfido.min.js`
+* If they exist, copy each of them and paste on https://rawgit.com
+* Open the JSFiddle and update its resources to the following:
+  * `https://cdn.rawgit.com/onfido/onfido-sdk-ui/<tag>/dist/style.css`
+  * `https://cdn.rawgit.com/onfido/onfido-sdk-ui/<tag>/dist/onfido.min.js`
+* Follow the migration notes and update the code if necessary
+* Test the happy path
+
```
Comment (hjheath) [inline release_guidelines.md]: Update the sample app too (possibly out of scope)

File: webpack.config.babel.js [hunk 4]
```diff
@@ -92,7 +96,11 @@ const basePlugins = [
     'ONFIDO_API_URL': CONFIG.ONFIDO_API_URL,
     'ONFIDO_SDK_URL': CONFIG.ONFIDO_SDK_URL,
     'SDK_VERSION': packageJson.version,
-    'WOOPRA_DOMAIN': `${DEV_OR_STAGING ? 'dev-':''}onfido-js-sdk.com`
+    'WOOPRA_DOMAIN': `${DEV_OR_STAGING ? 'dev-':''}onfido-js-sdk.com`,
+    'DESKTOP_SYNC_URL': CONFIG.DESKTOP_SYNC_URL,
+    'MOBILE_URL' : CONFIG.MOBILE_URL,
+    // Increment this with each release
```
Comment (hjheath) [inline webpack.config.babel.js]: Increment it how? Link to https://en.wikipedia.org/wiki/Base32 please

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-242-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://242-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**